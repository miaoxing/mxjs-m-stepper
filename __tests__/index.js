import Stepper from '..';
import {render} from '@testing-library/react';

describe('Stepper', () => {
  test('basic', () => {
    const result = render(<Stepper/>);
    expect(result.container).toMatchSnapshot();
  });
});
