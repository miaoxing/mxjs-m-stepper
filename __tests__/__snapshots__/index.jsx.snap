// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Stepper basic 1`] = `
<div>
  <taro-view-core
    class="mx-stepper"
  >
    <taro-button-core
      class="mx-stepper-minus"
    />
    <taro-input-core
      class="mx-stepper-input"
    />
    <taro-button-core
      class="mx-stepper-plus"
    />
  </taro-view-core>
</div>
`;
